### DISCUSSION TITLE
How to color block of text

### DISCUSSION THREAD

**Discussion by stalkerGH** [welpo/tabi#454] [Q&A] [answered]
I'd like to use different color in long text - one or several words. Or some cells in table. How can I do that?

**Comment by welpo** [comment, marked as answer]
You'll need to use HTML and [load custom CSS](https://welpo.github.io/tabi/blog/mastering-tabi-settings/#custom-css).

For example, in your article:

```md
This <span class="blue">is blue</span>
```

And load CSS:
```css
.blue {
    color: blue;
    font-weight: bold;
}
```

Or… you could disable CSP/allow inline styles and do something like:

```html
<p style="color: blue; font-size: 20px; text-align: center;">
    This is a paragraph with inline styles.
</p>
```

**Reply by stalkerGH** [reply]
Thank you, it was simple :)
